### PR TITLE
completed basic flow API with the example of profile fragment

### PR DESCRIPTION
**Pull request by malikdawar**
# Description #

Completed the basic flow of API with the example of profile fragment to demonstration purpose. 

Fixes (Trello card link)

## Type of change

Please check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* OS version:
* Hardware:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/src/main/java/com/malik/covid/App.kt [hunk 1]
```diff
@@ -2,12 +2,14 @@ package com.malik.covid
...
+import com.malik.covid.network.RetrofitClient
+import com.malik.covid.repository.Repository
+import com.malik.covid.utils.Const.BASE_URL
 import org.koin.android.ext.koin.androidContext
-import org.koin.androidx.viewmodel.dsl.viewModel
 import org.koin.core.context.startKoin
 import org.koin.dsl.module
 
-class App: Application() {
+class App : Application() {
```
Comment (joostheijkoop) [inline app/src/main/java/com/malik/covid/App.kt:12]: typo?

File: app/src/main/java/com/malik/covid/network/ApiInterface.kt [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 
+import com.malik.covid.models.dao.UserDao
+import retrofit2.http.Body
+import retrofit2.http.POST
+
 
 interface ApiInterface {
 
+    @POST("")
+    suspend fun userProfile(@Body userId: String?): UserDao?
```
Comment (joostheijkoop) [inline app/src/main/java/com/malik/covid/network/ApiInterface.kt:11]: I assume, this will go away again ;)

**Review by joostheijkoop — APPROVED** [review]
Lots of stubs, but otherwise LGTM

File: app/src/main/java/com/malik/covid/App.kt (hunk 1 above)
Comment (malikdawar) [reply app/src/main/java/com/malik/covid/App.kt:12]: Not the typo, just reformatted the code.

File: app/src/main/java/com/malik/covid/network/ApiInterface.kt (hunk 2 above)
Comment (malikdawar) [reply app/src/main/java/com/malik/covid/network/ApiInterface.kt:11]: This Pull request is the basic idea to call the API using MVVM architecture, the endpoint will be updated.

**Pull request merged**